compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/repro.expect.md

```diff
@@ -24,27 +24,41 @@ function Component(props) {
 ```javascript
 import { c as _c } from "react/compiler-runtime";
 function Component(props) {
-  const $ = _c(2);
+  const $ = _c(7);
```

Comment:
this is unfortunate, same as the other weird output change the only thing that could have caused this is that the DropManualMemo rewriting is now enabled for this pass, and whatever it's doing is causing these two reactive scopes to not be eligible for merging anymore. i'll follow-up on this. note that the output we get on this fixture is reflective of what we already would have seen internally, though, since we have validatePreserveManualMemo enabled